tomb-wasm/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 //! This crate contains modules which are compiled to WASM
+#![feature(let_chains)]
```

Comment:
Please don't use these. Part of the tech debt I'm going to have people working on while I'm gone is getting these crates compiling on stable rust and removing these non-standard-for-a-very-good-reason extensions to the syntax.